main/main.go

```diff
@@ -112,19 +110,14 @@ func initLogging() (*os.File, error) {
 		return nil, errors.Wrapf(err, "Failed to create/open %s", logfile)
 	}
 
-	/* Log UTC Time, Date, Time, (w/ microseconds), line number, and make message prefix come right
-	before the message
-	*/
-	loggerFlags := log.LUTC | log.Ldate | log.Ltime | log.Lmicroseconds | log.Lshortfile | log.Lmsgprefix
+	//Sample: [2020-08-18T20:29:16.079902Z] [1.0.0.0] [main.go:148] [INFO]: Test1
+	infoLogger.logger = log.New(io.MultiWriter(file, os.Stdout), "", 0)
+	warningLogger.logger = log.New(io.MultiWriter(file, os.Stderr), "", 0)
+	errorLogger.logger = log.New(io.MultiWriter(file, os.Stderr), "", 0)
 
-	// Sample out: 2020/07/31 21:47:19.153535 main.go:145: Version: 1.0.0.0 INFO: Hello World!
-	infoLogger = log.New(io.MultiWriter(file, os.Stdout), "Version: "+version+" INFO: ", loggerFlags)
-
-	// Sample out: 2020/07/31 21:47:19.153535 main.go:145: Version: 1.0.0.0 WARNING: Hello World!
-	warningLogger = log.New(io.MultiWriter(file, os.Stderr), "Version: "+version+" WARNING: ", loggerFlags)
-
-	// Sample out: 2020/07/31 21:47:19.153535 main.go:145: Version: 1.0.0.0 ERROR: Hello World!
-	errorLogger = log.New(io.MultiWriter(file, os.Stderr), "Version: "+version+" ERROR: ", loggerFlags)
+	infoLogger.initLogger("INFO")
```

Comment:
Use the constants in the `customLogger` module here (customLogger.infoOperation) and so on